src/operator/csv.rs

```diff
@@ -0,0 +1,110 @@
+//! Source operator that reads data from a CSV file.
```

Comment:
Does this really need to be part of the library itself? At very least it should be feature gated, most users won't need it and so don't need the dependencies it pulls in

Reply:
That's a good point.  It makes sense to keep non-core operators in separate crates.  Which by the way reminds me that we should move to crates.io now that the project is open source.

This particular operator we are using it in benchmarks that live in the same crate, so I don't think it can be easily separated. I am also not sure how to feature gate it so that benchmarks can still use it.  Any ideas?

Reply:
You'd pretty much just feature gate the relevant code, add `optional = true` to the `csv` dependency and then add `required-features = ["csv"]` to the benches that need it